interface/logKappa.h

```diff
@@ -0,0 +1,23 @@
+#ifndef logKappa_h
+#define logKappa_h
+
+template<typename T>
+T logKappa(const T x, const double &kappaHigh, const double &kappaLow) {
+    if (fabs(x) >= 0.5) return (x >= 0 ? kappaHigh : -kappaLow);
```

Comment:
```suggestion
    if (std::abs(x) >= 0.5) return (x >= 0 ? kappaHigh : -kappaLow);
```
Better to use functions from the C++ standard library and don't forget to `#include <cmath>` in this file to be sure to have the math functions.